core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -1976,6 +1976,49 @@ private[spark] object Utils extends Logging {
     }
   }
 
+  def makeQualified(defaultUri: URI, workingDir: Path, path: Path): Path = {
```

Comment:
This is a utility function. We normally need to add function comments and param description.